wordpress-mobile/WordPress-iOS
# [Domain Management] List Card UI

Fixes #21434 

## Testing Steps
1. Go to WordPressAppDelegate.swift
2. Comment out `willFinishLaunchingWithOptions` and `didFinishLaunchingWithOptions` functions. This is to hack SwiftUIPreview functionality.
3. Go to `DomainListCard.swift`
4. Scroll to bottom and find `DomainListCard_Previews` extension. 
5. Manipulate values on `DomainListCard` init and verify the changes on the preview.
6. Also toggle the `.environment(\.colorScheme, .light)` parameter between `light` & `dark`

## Screenshot References
| Active - Light | Action Required - Dark  |
| ------------- | ------------- |
| ![Screenshot 2023-09-21 at 14 24 38](https://github.com/wordpress-mobile/WordPress-iOS/assets/15968946/7ef0252b-9bfe-4efc-a1e2-cfb14470059f)  | ![Screenshot 2023-09-21 at 14 24 53](https://github.com/wordpress-mobile/WordPress-iOS/assets/15968946/12113156-7f12-40dd-be78-e4dc19763e2a)  |

## Regression Notes
1. Potential unintended areas of impact
N/A: All UI additions with no integration to the current flows. So the additions are isolated.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:


UI Changes testing checklist:

The missing options above can be tested once we integrate this to a screen. Right now we can test on Previews.
